jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamodb/Attributes.kt

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.dynamodb
+
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue
+import java.util.Base64
+
+typealias SearchResults = List<Map<String, DynamoAttribute<*>>>
+
+/**
+ * See:
+ * * https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_AttributeValue.html,
+ * * https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html
+ */
+sealed class DynamoAttribute<T>(val value: T) {
+    abstract val dataType: String
+
+    open fun stringRepresentation() = value.toString()
+}
+
+class StringAttribute(value: String) : DynamoAttribute<String>(value) {
+    override val dataType: String = "S"
+
+    override fun stringRepresentation(): String = '"' + value + '"'
```

Comment:
there's 3 different way you're making a `"` quoted string in this file lol

Reply:
cleaned up